src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -80,6 +81,13 @@ cdef class GlyphPosition:
         return self._hb_glyph_position.y_offset
 
 
+class BufferClusterLevel(IntEnum):
+    MONOTONE_GRAPHEMES = HB_BUFFER_CLUSTER_LEVEL_MONOTONE_GRAPHEMES
+    MONOTONE_CHARACTERS = HB_BUFFER_CLUSTER_LEVEL_MONOTONE_CHARACTERS
+    CHARACTERS = HB_BUFFER_CLUSTER_LEVEL_CHARACTERS
+    DEFAULT = HB_BUFFER_CLUSTER_LEVEL_DEFAULT
+
```

Comment:
I tried to move this class inside`Buffer` class below, so it does not need manual namespacing Buffer**.**ClusterLevel vs  BufferClusterLevel, but Cython does not allow that.